docs/ref/databases.txt

```diff
@@ -22,6 +22,12 @@ This file describes some of the features that might be relevant to Django
 usage. It is not intended as a replacement for server-specific documentation or
 reference manuals.
 
+.. note::
+
+   Most database backends require separate installation of client libraries and
+   Python modules in order to be used, as explained in their respective notes
+   sections. Django will not install these libraries or modules automatically.
+
```

Comment:
TBH I think this is covered by the per-DB sections. 

e.g. "Django supports PostgreSQL 9.6 and higher. `psycopg2`_ 2.5.4 or higher is required"